### DISCUSSION TITLE
Is it possible to specify the link section of static assets, such as fonts?

### DISCUSSION THREAD

**Discussion by markszente** [slint-ui/slint#5768] [Q&A] [answered]
In Rust, I can use the [`link_section`](https://doc.rust-lang.org/reference/abi.html#the-link_section-attribute) attribute to specify where static values will be placed into. Is it possible to have something like that with Slint? 
Essentially, I would like my font files to be placed in a specific flash section so that they don't have to be included with firmware updates.

**Comment by tronical** [comment, marked as answer]
We have an undocumented `SLINT_ASSET_SECTION` environment variable, which - if set - will place images and pre-rendered glyphs into the section the variable names. It literally results in the Slint compiler emitting `#[link_section = XXX]` :)

**Reply by markszente** [reply]
Thanks!
